src/tribol/utils/TestUtils.cpp

```diff
@@ -363,37 +371,29 @@ void TestMesh::setupContactMeshHex( int numElemsX1, int numElemsY1, int numElems
                   "Initial mesh configuration has a gap greater than the " << 
                   "element thickness in block 2.");
 
-   for (int iblk=0; iblk<2; ++iblk)
-   {
-      if (iblk == 0)
-      {
-         numElementsBlock1 = numElemsX1 * numElemsY1 * numElemsZ1;
-         numNodesBlock1 = (numElemsX1+1) * (numElemsY1+1) * (numElemsZ1+1);
-         this->numMortarFaces = numElemsX1 * numElemsY1;
-      }
-      else
-      {
-         numElementsBlock2 = numElemsX2 * numElemsY2 * numElemsZ2;
-         numNodesBlock2 = (numElemsX2+1) * (numElemsY2+1) * (numElemsZ2+1);
-         this->numNonmortarFaces = numElemsX2 * numElemsY2;
-      }
-   }
+   numElementsBlock1       = numElemsX1 * numElemsY1 * numElemsZ1;
+   numNodesBlock1          = (numElemsX1+1) * (numElemsY1+1) * (numElemsZ1+1);
+   this->numMortarFaces    = numElemsX1 * numElemsY1;
+   numElementsBlock2       = numElemsX2 * numElemsY2 * numElemsZ2;
+   numNodesBlock2          = (numElemsX2+1) * (numElemsY2+1) * (numElemsZ2+1);
+   this->numNonmortarFaces = numElemsX2 * numElemsY2;
 
-   this->numMortarNodes = numNodesBlock1;
-   this->numNonmortarNodes = numNodesBlock2;
+   this->numMortarNodes           = numNodesBlock1;
+   this->numNonmortarNodes        = numNodesBlock2;
    this->numNonmortarSurfaceNodes = (numElemsX2+1) * (numElemsY2+1);
-   this->numTotalNodes = numNodesBlock1 + numNodesBlock2;
-   this->numMortarElements = numElementsBlock1;
-   this->numNonmortarElements = numElementsBlock2;
-   this->numTotalElements = numElementsBlock1 + numElementsBlock2;
-
-   this->elConn1 = new int[ this->numNodesPerElement * this->numMortarElements ];
-   this->elConn2 = new int[ this->numNodesPerElement * this->numNonmortarElements ];
+   this->numTotalNodes            = numNodesBlock1 + numNodesBlock2;
+   this->numMortarElements        = numElementsBlock1;
+   this->numNonmortarElements     = numElementsBlock2;
+   this->numTotalElements         = numElementsBlock1 + numElementsBlock2;
+   this->numTotalFaces            = this->numNonmortarFaces + this->numMortarFaces;
+
+   this->elConn1   = new int[ this->numNodesPerElement * this->numMortarElements ];
+   this->elConn2   = new int[ this->numNodesPerElement * this->numNonmortarElements ];
    this->faceConn1 = new int[ this->numNodesPerFace * this->numMortarFaces ];
    this->faceConn2 = new int[ this->numNodesPerFace * this->numNonmortarFaces ];
-   this->x = new real[ this->numTotalNodes ];
-   this->y = new real[ this->numTotalNodes ];
-   this->z = new real[ this->numTotalNodes ];
+   this->x = new double[ this->numTotalNodes ];
+   this->y = new double[ this->numTotalNodes ];
+   this->z = new double[ this->numTotalNodes ];
```

Comment:
Why is this a `double` vs a `real`?  If the plan is to get rid of tribol's data types in favor of the standard C++ ones, that's fine (I'm not sure if there's a reason to support single precision `float`?).   It would be good to standardize this throughout the codebase though (except when non-standard types like `axom::IndexType` or `HYPRE_BigInt` are called for).

Reply:
This is likely an old convention that I have mangled. I'll create an issue and we can do a quick PR.